### PR TITLE
feat: nft search

### PR DESCRIPTION
**Pull request by DanielSinclair**
Fixes BX-####
Figma link (if any):

## What changed (plus any additional context for devs)
- added `My NFTs` search section
- added downranked NFTs search with token id badge with `NFTRow` component
- added `UniqueAsset` handling to `NFTIcon`
- added `NFT_TOKEN_DETAIL` route handling for command k
- added `useSearchableNFTs` search hook
- renamed `View NFTs` web profiles command to `View Profile`

## Screen recordings / screenshots

Screenshot 2023-12-18 at 2 38 09 PM
Screenshot 2023-12-18 at 2 38 22 PM
Screenshot 2023-12-18 at 2 40 00 PM

## What to test
- Search for NFTs in Command K
- Is performance impacted by use of `useNFTs()` fetching? Loading thumbnails in command k?

### CODE REVIEW

File: src/design-system/components/Box/Box.tsx [hunk 1]
```diff
@@ -58,9 +58,9 @@ export const Box = forwardRef(function Box(
   },
   ref,
 ) {
-    let hasBoxStyles = false;
```
Comment (derHowie) [inline src/design-system/components/Box/Box.tsx:61]: don't really care, but curious what causing the formatting diff here

File: src/entries/popup/components/CoinIcon/CoinIcon.tsx [hunk 2]
```diff
@@ -262,21 +263,28 @@ export const NFTIcon = ({
   size,
   badge = false,
 }: {
-  asset: ParsedAsset;
+  asset: ParsedAsset | UniqueAsset;
   size: keyof typeof nftRadiusBySize;
   badge?: boolean;
 }) => {
-  const chainId = asset.chainId;
+  const chainId = 'chainId' in asset ? asset.chainId : undefined;
```
Comment (derHowie) [inline src/entries/popup/components/CoinIcon/CoinIcon.tsx:270]: I've never done this to deal with a fork in type like this. v nice

**Review by derHowie — APPROVED** [review]
looks good, nice job

File: src/design-system/components/Box/Box.tsx (hunk 1 above)
Comment (DanielSinclair) [reply src/design-system/components/Box/Box.tsx:61]: I think a few style issues were introduced here. Prettier ran on the git commit that resolved them, I guess. https://github.com/rainbow-me/browser-extension/pull/1180

**Review by greg-schrammel — APPROVED** [review]
👑

**Pull request merged**